tests/build/build.go

```diff
@@ -248,6 +288,31 @@ var _ = framework.BuildSuiteDescribe("Build service E2E tests", Label("build", "
 					return true
 				}, timeout, interval).Should(BeTrue(), "timed out when waiting for the PipelineRun to finish")
 			})
+			It("image repo and robot account created successfully", func() {
+
+				component, err := f.AsKubeAdmin.HasController.GetHasComponent(componentName, testNamespace)
+				Expect(err).ShouldNot(HaveOccurred(), "could not get component %s in the %s namespace", componentName, testNamespace)
+
+				annotations := component.GetAnnotations()
+				imageRepoName, err = build.GetQuayImageName(annotations)
+				Expect(err).ShouldNot(HaveOccurred(), "failed to read image name")
+
+				imageExist, err := build.DoesImageRepoExistInQuay(imageRepoName)
+				Expect(err).ShouldNot(HaveOccurred(), "failed while checking if image repo exists in quay")
+				Expect(imageExist).To(BeTrue(), "quay image does not exists")
+
+				robotAccountName = build.GetRobotAccountName(imageRepoName)
+				robotAccountExist, err := build.DoesRobotAccountExistInQuay(robotAccountName)
+				Expect(err).ShouldNot(HaveOccurred(), "failed while checking if robot account exists in quay")
+				Expect(robotAccountExist).To(BeTrue(), "quay robot account does not exists")
```

Comment:
more info to these errors as well please

Reply:
Done